event-subscription/service/example-subscriber.yaml

```diff
@@ -29,7 +30,7 @@ spec:
     metadata:
       labels:
         app: event-email-service
-        example: event-bus-service-subscription
+        example: event-subscription-service
```

Comment:
It is more of a subscriber than a susbcription-service.

Reply:
Applies to other subscription yamls also.

Reply:
I tried to label all resources for an example with the example label using the name of the example. Here the folder name is event-subscription-service.